src/commands/myanimelist.ts

```diff
@@ -0,0 +1,91 @@
+import * as Discord from "discord.js";
+import * as _ from "lodash";
+import { getAnimeManga, MediumType } from "../api/myanimelist";
+
+interface Field {
+  name: string;
+  value: string;
+}
+
+export function getAnimeAndMangaInfo(
+  channel: Discord.TextChannel,
+  command: MediumType,
+  args: string[]
+): Promise<Discord.Message | Discord.Message[]> {
+  if (!args.length) {
+    return channel.send(
+      `\`\`\`You didn't provide enough arguments: requires: <!anime/manga anime_title>, spaced anime/manga names require: <!anime/manga "anime/manga_title">\`\`\``
+    );
+  } else {
+    const [title] = args;
+
+    getAnimeManga(command, title)
```

Comment:
I think for your use case, await syntax is better here and deal with the error handling in a simple try catch at the entry point to ur application rather than having a .catch on every api call you're making, because you're not doing any specific error logic other than logging the error and sending back the error to the channel